### ISSUE TITLE
Feature request: Reach out Chrome team

### ISSUE THREAD

**Issue by tinovyatkin** [node-fetch/node-fetch#839]
Google Chrome includes `Copy as Node.js fetch` context menu item at Network requests:
![Screenshot 2020-05-25 at 18 14 04](https://user-images.githubusercontent.com/5350898/82845771-c5051280-9eb3-11ea-9eb1-b31cbcc74c11.png)

however, it creates invalid code by including unsupported options:
```js
fetch("https://github.com/orgs/node-fetch/hovercard", {
  "headers": {
    "accept": "*/*",
    "accept-language": "en-US,en;q=0.9,ru-RU;q=0.8,ru;q=0.7",
    "sec-fetch-dest": "empty",
    "sec-fetch-mode": "cors",
    "sec-fetch-site": "same-origin",
  },
  "referrer": "https://github.com/node-fetch/node-fetch/issues",
  "referrerPolicy": "no-referrer-when-downgrade",
  "body": null,
  "method": "GET",
  "mode": "cors"
});
```

Can somebody read Chromium/Google developers?

**Comment by xxczaki** [comment]
cc @addy

**Comment by xxczaki** [comment]
Wow, I did not know about this feature. I will try and contact some Chrome developers on Twitter tomorrow.

**Comment by NotMoni** [comment]
![Screen Shot 2020-05-25 at 6 51 15 PM](https://user-images.githubusercontent.com/40552237/82846813-b9681a80-9eb8-11ea-867e-7d11a2383c3d.png)

**Same thing in Firefox ?**

![Screen Shot 2020-05-25 at 6 51 27 PM](https://user-images.githubusercontent.com/40552237/82846819-bec56500-9eb8-11ea-8878-36e1f1dedf99.png)

**Comment by xxczaki** [comment]
Yes, but I think this was made to be used in the browser and not Node.js :/

**Comment by NotMoni** [comment]
👍

**Comment by TimothyGu** [comment]
Looks like all it does right now is copying the cookies as well as other things. See https://crbug.com/1029826. @mathiasbynens?

**Comment by mathiasbynens** [comment]
@jschfflr, could you please take a look?

**Comment by jschfflr** [comment]
Hi, thanks for reaching out. I contributed the feature mainly to include cookies in the headers because copy as fetch relies on `credentials: 'include'`.
Could you elaborate on which options you think should be included and which not?

**Comment by JefferyHus** [comment]
@tinovyatkin ☝️

**Comment by tinovyatkin** [comment]
@jschfflr Thank you for getting back to us!
`node-fetch` doesn't support `referrer`, so, can this value be moved into the appropriate header?
The library also doesn't use  `referrerPolicy` and `mode` options, so, can they be omitted?

The POST body (for JSON requests) is looking a little bit weird, but it's OK for generated code.

Thank you!

**Comment by jimmywarting** [comment]
I would like to ask the question: can we find any value into adding credentials, referrer, referrerPolicy & mode into node-fetch? wouldn't say unsupported properties are't completely useless they will just be ignored. We might support them one day doe.

- **credentials**: #274, #49 maybe useful for some cookie-jar addon or proxy later? maybe you don't want to send the cookie for static files or something like that.
- **referrerPolicy** maybe someone who uses a proxy wants to exclude referrer header always?
- **mode** well, we don't live under the same rules... so pretty useless?

**Comment by jschfflr** [comment]
@jimmywarting That was also my line of thought. This option was mainly intended to include cookies in the header and not specifically targeted to node-fetch. There might be other fetch implementation (for example isomorphic-fetch) that benefit from having those options included.
Also, I think it's easier to remove unwanted options from the copied object than to add them - that's also the main reason I added the cookie header in there in the first place. What do you think?

Do you maybe have a better idea how this option should be named to reduce confusion?

**Comment by jimmywarting** [comment]
I'm fine with the way it is. Don't really see it as "invalid" code, typescript might complain about unknown properties but that's it... Nothing harmful. 

The cookie thing is useful doe! :+1: 

One thing i would like doe is to get rid of unnecessary default properties. Even for "copy as browser fetch" think it creates lots of boiler code. Method get is unnecessary, body null is unnecessary etc
